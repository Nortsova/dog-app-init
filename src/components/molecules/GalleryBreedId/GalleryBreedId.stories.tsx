import type { Meta, StoryObj } from '@storybook/react';
import { GalleryBreedId } from '.';

const meta = {
  title: 'Molecules/GalleryBreedId',
  component: GalleryBreedId
} satisfies Meta<typeof GalleryBreedId>;

export default meta;
type Story = StoryObj<typeof meta>;

export const Primary: Story = {
  args: {
    breedId: '1'
  }
};
