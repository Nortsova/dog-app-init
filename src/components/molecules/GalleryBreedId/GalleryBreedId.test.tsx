import { composeStories } from '@storybook/react';
import * as stories from './GalleryBreedId.stories';
import { testSnapshot } from '../../../test/utils/snapshot.utils';

describe('GalleryBreedId', () => {
  const { Primary } = composeStories(stories);
  testSnapshot(<Primary />);
});
