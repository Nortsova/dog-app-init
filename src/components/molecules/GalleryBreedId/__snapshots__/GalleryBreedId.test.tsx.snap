// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`GalleryBreedId should match snapshot Component 1`] = `
.emotion-0 {
  display: -webkit-box;
  display: -webkit-flex;
  display: -ms-flexbox;
  display: flex;
  gap: 100px;
  padding-top: 50px;
}

<div>
  <div
    class="MuiBox-root emotion-0"
  >
    <div>
      loading image
    </div>
    <div>
      loading breed
    </div>
  </div>
</div>
`;
